core/chains/cosmos.go

```diff
@@ -18,70 +18,131 @@ import (
 
 const BONDED = "BOND_STATUS_BONDED"
 
+func Cosmos() (int, error) {
+	validatorURL := "https://proxy.atomscan.com/cosmoshub-lcd/cosmos/staking/v1beta1/validators?page.offset=1&pagination.limit=500&status=BOND_STATUS_BONDED"
+	poolURL := "https://proxy.atomscan.com/cosmoshub-lcd/cosmos/staking/v1beta1/pool"
+
+	return FetchCosmosSDKNakaCoeff("cosmos", validatorURL, poolURL)
+}
+
 type cosmosResponse struct {
 	Validators []struct {
-		Status string `json:"status"`
-		Tokens string `json:"tokens"`
+		OperatorAddress string `json:"operator_address"`
```

Comment:
we can probably remove some of these fields in the response as we don't use/need them in our calculations. eg: `Description`, `Commission` etc are probably not required.

Reply:
I kept them in incase at any point someone wanted to add some additional detail or context but yeah it's bloat anyway.